14-eco.Rmd

```diff
@@ -1,4 +1,6 @@
-# Ecological case study
+# (PART) Geocomputation in the wild {-}
```

Comment:
Note the more interesting part title. More exciting than Part I and II. The distinction is that these latter chapters are more 'wild' in terms of difficulty level is the thinking.

Reply:
Good thinking!